astropy/io/fits/hdu/table.py

```diff
@@ -248,9 +248,28 @@ class _TableBaseHDU(ExtensionHDU, _TableLikeHDU):
     """
 
     def __init__(self, data=None, header=None, name=None, uint=False):
+        """
```

Comment:
Parameters go in the Class docstring, not in `__init__`.  Also no line whitespace between params.